### PR TITLE
Panlint cleanup

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
@jrha i've tagged this 17.7, feel free to bump this

**Comment by jrha** [comment]
Also removes aii-freeipa?

**Comment by stdweird** [comment]
yes. should i make separate PR?

**Comment by jrha** [comment]
Yes please. It's probably best if this one is just lint cleanup.

**Comment by stdweird** [comment]
@jrha removed and new PR made. tagged for 17.8 also.

File: aii-ks/src/main/pan/quattor/aii/ks/schema.pan [hunk 1]
```diff
@@ -75,50 +75,50 @@ type structure_ks_mail = {
...
-    "mail"      : structure_ks_mail
-    "bootproto"	: string with match (SELF, "static|dhcp")
-    "keyboard"	: string = "us"
-    "node_profile"	: type_absoluteURI
-    "rootpw"	: string
+    "langsupport" ? string [] = list ("en_US.UTF-8")
+    "logging" ? structure_ks_logging
+    "mouse" ? string
+    "bootproto" : string with match (SELF, "static|dhcp")
+    "mail" : structure_ks_mail
```
Comment (jrha) [inline aii-ks/src/main/pan/quattor/aii/ks/schema.pan]: any particular reason these lines switched places?

Comment (stdweird) [reply aii-ks/src/main/pan/quattor/aii/ks/schema.pan]: no. i probably moved the `mail` around a bit and then decided to put it back in the wrong place 😄 do you want me to change it?

Comment (jrha) [reply aii-ks/src/main/pan/quattor/aii/ks/schema.pan]: Doesn't really matter, just confuses the history slightly.

**Pull request merged**